src/tss2-tcti/tcti-device.c

```diff
@@ -393,7 +421,11 @@ Tss2_Tcti_Device_Init (
     memset (&tcti_common->header, 0, sizeof (tcti_common->header));
     tcti_common->locality = 3;
 
+#ifdef __VXWORKS__
```

Comment:
I think, instead of having an ifdef, we could just use the 3-parameter version for VxWorks as well as other POSIX systems. We should just add a comment that explains why we use the 3-parameter version.

Reply:
except the 3-parameter issue, the flag of open() is different. VxWorks requires O_RDWR without O_NONBLOCK, but other OS require O_RDWR | O_NONBLOCK.